src/game/game.cpp

```diff
@@ -85,6 +85,60 @@ namespace InternalGame {
 		}
 	}
 
+	bool playerCanUseItemOnHouseTile(Player* player, Item* item) {
+		if (!player || !item) {
+			return false;
+		}
+
+		auto itemTile = item->getTile();
+		if (!itemTile) {
+			return false;
+		}
+
+		if (HouseTile* houseTile = dynamic_cast<HouseTile*>(itemTile)) {
+			House* house = houseTile->getHouse();
+			// If not invited return false
```

Comment:
thought: this comment just re-states what the code is doing, I think we can delete it

Reply:
actually same for the other one line comments in this function here